README.md

```diff
@@ -11,29 +11,47 @@
 
 💡📊A prototype data dashboard for any FHIR server. Spin this up in front of your server so people can get a quick visual understanding of what's in this server.
 
-## Development
+## Getting Started
 
-This application utilizes `create-react-app`. To run it, install the dependencies
-and start the server:
+If you want to run the dashboard app and don't want to install
+dependencies directly on your machine, you can spin up the Dockerized dashboard
```

Comment:
Who do we think is using the Docker version of the app ? Just trying to think through if it makes sense to set the REACT_APP_FHIR_APP variable, or if the people using the app  aren't KF users so they can't access the data, since they need the tunnel? So maybe HAPI would be a better default, or just no default and the users pick via the UI?

Reply:
I think the primary use case for this is spinning up a docker-compose stack with the fhir server and app on your machine. That's why the Dockerfile sets the env vars to point to local host. Later on the docker stack would probably include other pieces of the CDF pipeline (e.g. ingest warehouse, maybe a FHIR staging server). 

A secondary use case would be someone who wants to demo the dashboard with a server that doesn't require tunneling into the KF AWS VPC. This secondary use case will likely go away in the long run once the app and server are deployed to production and made public (with proper authn/authz controls in place).

Reply:
Oh forgot to add - I'm also using this to deploy the app on our EC2 server where the FHIR services are running.

Reply:
Okay! Dashboard seems to work for me, the server at localhost:8000 doesn't though - is that expected?